CITATION.rst

```diff
@@ -0,0 +1,36 @@
+..
+   This file is generated by team2.py using the citations.rst_t template and team.ini.
+      DO NOT EDIT DIRECTLY
+   To add your name and/or institution edit the relevant sections of
+      docs/source/team.ini
```

Comment:
should autogenerated files be in the repo?

Reply:
In general no, but CITATION.rst and AUTHORS.rst exist to enable Github functionality and must be committed.